tests/gui/test_app.py

```diff
@@ -4,12 +4,27 @@
 from sleap.gui.app import MainWindow
 from sleap.gui.commands import *
 
+# Online bulletin if there is an ImportError (for MacOS)
+online_bulletin = False
+try:
+    from sleap.gui.dialogs.bulletin import BulletinWorker
+except ImportError:
+    online_bulletin = True
+
```

Comment:
Import this from `sleap.gui.app` so we cover that functionality rather than duplicating it here